transit_odp/timetables/tests/test_csv.py

```diff
@@ -42,18 +42,16 @@ def test_service_in_bods_but_not_in_otc():
         assert row["Published Status"] == "Published"
         assert row["OTC Status"] == "Unregistered"
         assert row["Scope Status"] == "In Scope"
-        assert row["DQ Score"] == "33%"
-        assert row["BODS Compliant"] == "NO"
-        assert row["XML Filename"] == txc_file_attributes.filename
-        assert row["Last Updated Date"] == dataset.live_revision.published_at
+        assert row["XML:Filename"] == txc_file_attributes.filename
         assert (
-            row["National Operator Code"] == txc_file_attributes.national_operator_code
+            row["XML:National Operator Code"]
```

Comment:
There are 41 or so columns in the catalogue but the tests are verifying 30 column values?